packages/manager/modules/pci/src/projects/project/sidebar/project-list/project-list.html

```diff
@@ -41,6 +42,15 @@
                 class="text-white"
                 data-translate="cloud_sidebar_projectlist_title"
             ></h4>
+            <button
```

Comment:
Shouldn't we use a [`<oui-button variant="seconday">`](https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-buttons-webcomponent-secondary--default) instead?

Reply:
These classes are overridden in the light theme and the colors are reversed. Hence, we will have to use primary to get a white button

Reply:
Thank you for the update.